workalendar/america/chile.py

```diff
@@ -1,45 +1,103 @@
 from datetime import date
 
-from ..core import WesternCalendar, MON, TUE, WED, FRI
+from ..core import WesternCalendar, MON, TUE, WED, THU, FRI, SAT
+from ..astronomy import solar_term
 from ..registry_tools import iso_register
 
 
+# https://www.feriados.cl
+# http://www.feriadoschilenos.cl
+
 @iso_register('CL')
 class Chile(WesternCalendar):
     "Chile"
     FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
         (5, 21, "Navy Day"),
-        (6, 29, "Saint Peter and Saint Paul"),
-        (7, 16, "Our Lady of Mount Carmel"),
         (9, 18, "National holiday"),
         (9, 19, "Army holiday"),
-        (10, 12, "Columbus Day"),
         (12, 31, "Banking Holiday"),
     )
     # Civil holidays
+    # Labor day (Law 2.200 and Law 18.018)
     include_labour_day = True
+
     # Christian holidays
+    # Holy Week (Law 2.977)
     include_good_friday = True
     include_easter_saturday = True
+    # Assumption (Law 2.977)
     include_assumption = True
+    # All Saints (Law 2.977)
     include_all_saints = True
+    # Immaculate Conception (Law 2.977)
     include_immaculate_conception = True
 
+    
     def get_variable_days(self, year):
         days = super().get_variable_days(year)
-        september_17 = date(year, 9, 17)
-        if september_17.weekday() == MON:
-            days.append((september_17, '"Bridge" holiday'))
-        september_20 = date(year, 9, 20)
-        if september_20.weekday() == FRI:
-            days.append((september_20, '"Bridge" holiday'))
-
-        reformation_day = date(year, 10, 31)
-        if reformation_day.weekday() == WED:
-            reformation_day = date(year, 11, 2)
-        elif reformation_day.weekday() == TUE:
-            reformation_day = date(year, 10, 27)
-
-        days.append((reformation_day, "Reformation Day"))
-
-        return days
+
+
+        # Indigenous Peoples (Law 21.357)
+        indigenous_peoples_day = date(year, 6, 21)
+        if year == 2021:
+            days.append((indigenous_peoples_day, 'Indigenous Peoples Day'))
+        elif year > 2021:
+            june_solstice = solar_term(year, 90, 'America/Santiago')
```

Comment:
I wonder if the June solstice could also be calculated directly with Skyfield, as in: https://rhodesmill.org/skyfield/almanac.html#the-seasons

I'm not sure which one is faster, it would need thourough tests.
Don't bother changing this here, I'll make a separate branch.

Reply:
All right